File: FeedleDataTier/FeedleDataTier/Network/RespondToFriendRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class RespondToFriendRequest : Request
    {
        [JsonPropertyName("respondStatus")]
        public bool RespondStatus { get; set; }
        [JsonPropertyName("friendRequestNotification")]
        public FriendRequestNotification FriendRequestNotification { get; set; }
        public RespondToFriendRequest(bool respondStatus, FriendRequestNotification friendRequestNotification) : base(RequestType.RespondToFriendRequest)
        {
            FriendRequestNotification = friendRequestNotification;
            RespondStatus = respondStatus;
        }
    }
}

File: Backend/src/main/java/com/feedle/feedleapi/Networking/RespondToFriendRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.FriendRequestNotification;

public class RespondToFriendRequest extends Request {
    private boolean respondStatus;
    private FriendRequestNotification friendRequestNotification;

    public RespondToFriendRequest(boolean respondStatus, FriendRequestNotification friendRequestNotification) {
        super(RequestType.RespondToFriendRequest);
        this.respondStatus = respondStatus;
        this.friendRequestNotification = friendRequestNotification;
    }

    public FriendRequestNotification getFriendRequestNotification() {
        return friendRequestNotification;
    }
}
